package.json

```diff
@@ -74,6 +74,7 @@
     "react-twitter-embed": "^3.0.3",
     "react-vertical-timeline-component": "^3.3.0",
     "react-visibility-sensor": "^5.1.1",
+    "sharp": "^0.28.3",

```

Comment:
We don't need this new package. So, please revert this change and the changes to the package-loock.json file.

Reply:
without this package, it couldn't run on my local machine.